druid/src/widget/list.rs

```diff
@@ -37,6 +39,24 @@ impl<T: Data> List<T> {
             children: Vec::new(),
         }
     }
+
+    /// When the widget is created or the data changes, create or remove children as needed
+    ///
+    /// Returns `true` if children were added or removed.
+    fn update_child_count(&mut self, data: &impl ListIter<T>, _env: &Env) -> bool {
+        let len = self.children.len();
+        match len.cmp(&data.data_len()) {
```

Comment:
Honestly not a fan of this `Ordering` nonsense, but my hidebound C way of doing things probably shouldn't override consensus from the Rust community in the form of this clippy lint. It partly seems ugly that the comparison is repeated, with different form, so it requires reasoning to see that it's the same comparison.

Reply:
I love Clippy. We have always been at war with Oceania.